src/main/java/com/dataloom/organizations/controllers/OrganizationsController.java

```diff
@@ -98,7 +104,7 @@ public Void updateTitle( @PathVariable( ID ) UUID organizationId, @RequestBody S
     @Override
     @PutMapping(
         value = ID_PATH + DESCRIPTION,
-        consumes = MediaType.TEXT_PLAIN_VALUE )
+        consumes = MediaType.APPLICATION_JSON_VALUE )
```

Comment:
`@RequestBody String description` is still only a `String`, not a JSON object, so does this need to change to `APPLICATION_JSON_VALUE`?